source/common/runtime/runtime_impl.cc

```diff
@@ -38,6 +42,21 @@ void countDeprecatedFeatureUseInternal(const RuntimeStats& stats) {
   stats.deprecated_feature_seen_since_process_start_.inc();
 }
 
+#ifdef ENVOY_ENABLE_QUIC
+void refreshQuicReloadableFlags(const Snapshot::EntryMap& flag_map) {
```

Comment:
Worth some release notes and docs for this functionality?  It's not discoverable otherwise.

Reply:
I added to do to document this behavior in the general Envoy QUIC doc.